lte/gateway/c/oai/tasks/nas/esm/sap/esm_recv.c

```diff
@@ -753,7 +754,13 @@ esm_cause_t esm_recv_activate_default_eps_bearer_context_accept(
   if (rc != RETURNerror) {
     esm_cause = ESM_CAUSE_SUCCESS;
   }
```

Comment:
This is not part of your diff, but is this section needed? On line 703, esm_cause is set to ESM_CAUSE_SUCCESS by default. The function call esm_proc_default_eps_bearer_context_accept(emm_context, ebi, &esm_cause) modifies it conditionally. I think if rc != RETURNok condition should be checked and if true, an error cause should be returned.

Reply:
fixed